src/sbvr-api/permissions.ts

```diff
@@ -1527,11 +1507,6 @@ export const resolveApiKey = async (
 	// TODO: Consider making tx the second argument in the next major
```

Comment:
@thgreasi ?

Reply:
@Page- that's just imo that `req, tx, paramName` makes more sense, since both req & tx provide context, while `param` is the real arguments. I'm fine with it as is though.